web/templates/non_editable_user_group.hbs

```diff
@@ -1,11 +0,0 @@
-{{#with user_group}}
-<div class="user-group white-box ntm" id="{{id}}">
-    <h4>
-        <span class="name" data-placeholder="{{t 'Name' }}">{{name}}</span>
-        —
-        <span class="description" data-placeholder="{{t 'Description' }}">{{description}}</span>
-    </h4>
-    <p class="subscribers">{{t 'Subscribers' }}</p>
-    <div class="pill-container not-editable" data-group-pills="{{id}}"></div>
-</div>
-{{/with}}
```

Comment:
This came with a test in `templates.js`. I think we might still implement the feature that used it but I'm not really a fan of it being a separate template rather than an option within an existing template, so probably makes sense to delete.

Reply:
The entire templates test was deleted in 7ec5fbab2e949914f93f6be081a8c51c13e97085 (#13886).